js/exporter/jspdf/draw_pdf.js

```diff
@@ -0,0 +1,102 @@
+import { isDefined } from '../../core/utils/type';
+import { drawTextInRect, drawRect } from './pdf_utils';
+import { extend } from '../../core/utils/extend';
+
+function drawPDF(doc, rows, options) {
```

Comment:
Maybe "drawPDF" -> "drawRow" because this method implements rendering for table rows.
And also make the file name as "draw_pdf_table.js" like in previous v2

Reply:
Yep, renamed drawPdf -> drawRows, because we exporting and drawing rows